docs/development/deployment.md

```diff
@@ -22,6 +22,13 @@ During the checkin before the end of the [current milestone](https://github.com/
 
 Note: we auto deploy the master branch to our *development environment*: [https://testpilot.dev.mozaws.net](https://testpilot.dev.mozaws.net)
 
+## Merge l10n ##
+
+Before tagging the release, merge the l10n branch into the master branch. These commands assume the remote named `mozilla` points at the mozilla repository on github.
+
+`git checkout -b l10n mozilla/l10n && git pull mozilla l10n && git checkout master && git merge l10n && git push mozilla master`
```

Comment:
I think you only need `git co -b` the first time, because it sets up branch tracking. Then `git co l10n && git pull mozilla l10n && ...` should work in future merges.